### DISCUSSION TITLE
Line-clamp with large text gets cut-off

### DISCUSSION THREAD

**Discussion by mohamedcherifmo** [tailwindlabs/tailwindcss#6677] [Help]
**What version of Tailwind CSS are you using?**
Tailwind 3.0

**What build tool (or framework if it abstracts the build tool) are you using?**

Tailwind Play

**What browser are you using?**

Edge

**Reproduction URL**

(https://play.tailwindcss.com/W6SR7G3PJL)
.

**Describe your issue**

Refer to the "g" in the word good, it's cut-off since the line-height is too small for the line-clamp. If line-height is inherit it would function normally

**Comment by adamwathan** [comment]
Hey! If you need to change the line-height you can just override it with a utility.

**Reply by mohamedcherifmo** [reply]
yes but there is no line height inherit in the default options (https://tailwindcss.com/docs/line-height), correct?
and should the default line-height cater for clamp font-size for clamp?

**Reply by simonmaass** [reply]
i have the same problem 

<img width="158" alt="image" src="https://github.com/user-attachments/assets/4cf7861e-bc63-4190-a780-faa58e931b4a" />

**Comment by rozsazoltan** [comment]
Related: [Descenders of letters get cropped out when using -webkit-line-clamp with reduced line-height](https://stackoverflow.com/a/79689030/15167500)

## v4.3

* https://github.com/tailwindlabs/tailwindcss/pull/20005
* https://github.com/tailwindlabs/tailwindcss/discussions/6677#discussioncomment-17134387

## v4.2

### `padding-bottom`

```css
@utility line-clamp-* {
  padding-bottom: calc(0.2lh - 0.035em);
}
```

* https://play.tailwindcss.com/lj7DQoMR8K?file=css

### `line-height`

```css
@utility line-clamp-* {
  line-height: var(--tw-leading, revert);
}
```

* https://play.tailwindcss.com/ViPoSGFh8q?file=css

**Reply by simonmaass** [reply]
@rozsazoltan the @utility doesnt work... both play examples dont work - or am i missing something?

**Reply by wongjn** [reply]
Those Play examples used unintended syntax that was removed in more recent versions of Tailwind. Here are the respective examples updated:

- https://play.tailwindcss.com/H0997GwL86?file=css
- https://play.tailwindcss.com/BYqPlkZyvT?file=css

**Reply by rozsazoltan** [reply]
* https://github.com/tailwindlabs/tailwindcss/pull/20005

**Comment by simonmaass** [comment]
is there any official workaround or solution for this?
